elasticgraph-graphql/lib/elastic_graph/graphql/filtering/filter_value_set_extractor.rb

```diff
@@ -134,6 +136,31 @@ def filter_value_set_for_any_of(filter_hashes, target_field_path_parts, traverse
           end
         end
 
+        # Determines the set of filter values for an `all_of` clause, which is used for ANDing multiple filters together.
+        def filter_value_set_for_all_of(filter_hashes, target_field_path_parts, traversed_field_path_parts, negate:)
+          # all_of: [] => match all values
+          if filter_hashes.empty?
+            return negate ? @empty_set : @all_values_set
```

Comment:
The build is failing due to one of the branches here missing test coverage. Would be good to add it.